mkdocs.yml

```
@@ -0,0 +1,102 @@
+site_name: SciLifeLab Architecture Board
```

I think this page should be about the architecture, not about us

```suggestion
site_name: SciLifeLab Architecture
```